src/Presentation/ConfigEditPageTextBuilder.php

```diff
@@ -124,4 +49,31 @@ private function getExampleContents(): string {
 		return $example;
 	}
 
+	private function getItemTypesData(): array {
+		$itemTypes = $this->config->getItemTypes();
+		if ( $itemTypes === [] ) {
+			return [];
+		}
```

Comment:
I think in cases like this it is not necessary to have an early exit. If the logic is based on an array loop, then the loop will do nothing and return an empty array anyway. The early exit suggests the empty array needs to be handled explicitly.

This can also be seen if the logic here was replaced by something like:
```php
return array_map(
    fn( $itemType ) => $this->getItemTypeData( $itemType ),
    $itemTypes
);
```